README.md

```diff
@@ -112,16 +112,33 @@ We recommend that you use [luarocks](https://luarocks.org/) to install APISIX, a
 
 ##### Install APISIX
 
+APISIX is installed by running the following commands in your terminal.
+
+> via curl
+
+```shell
+sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/iresty/apisix/master/utils/install-apisix.sh)"
+```
+
+> Manual inspection
+
+It's a good idea to inspect the install script from projects you don't yet know. You can do that by downloading the install script first, looking through it so everything looks normal, then running it:
```

Comment:
It's a good idea to inspect the installation script from projects you don't know yet. You can do that by downloading the installation script first, looking through it so everything looks normal, then running it: